From 9b410fbc5a7ada493a0d5403760b1717462a4089 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Wed, 2 Mar 2016 15:47:43 +0100
Subject: [PATCH 782/827] net: ethernet: xilinx: Reduce scope of frag variable

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Frag is used only in one for loop that's why scope of usage can be
reduced.

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Reviewed-by: Kedareswara rao Appana <appanad@xilinx.com>
(cherry picked from commit c6058af90e42bb429b3abfd78ca3ed427745b730)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/xilinx/xilinx_axienet_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_axienet_main.c b/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
index aa44a8d..76d6ae1 100644
--- a/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
+++ b/drivers/net/ethernet/xilinx/xilinx_axienet_main.c
@@ -799,7 +799,6 @@ static int axienet_start_xmit(struct sk_buff *skb, struct net_device *ndev)
 	u32 num_frag;
 	u32 csum_start_off;
 	u32 csum_index_off;
-	skb_frag_t *frag;
 	dma_addr_t tail_p;
 	struct axienet_local *lp = netdev_priv(ndev);
 	struct axidma_bd *cur_p;
@@ -883,6 +882,7 @@ static int axienet_start_xmit(struct sk_buff *skb, struct net_device *ndev)
 
 	for (ii = 0; ii < num_frag; ii++) {
 		u32 len;
+		skb_frag_t *frag;
 
 		++lp->tx_bd_tail;
 		lp->tx_bd_tail %= TX_BD_NUM;
-- 
2.9.3

